packages/conf/__typetests__/index.test-d.tsx

```diff
@@ -33,19 +33,19 @@ expectAssignable<LinguiConfig>({
     tsExperimentalDecorators: false,
   },
   fallbackLocales: {} as FallbackLocales,
-  format: "po",
+  format: "po" as const,
```

Comment:
The behaviour is the sam his discussed here: https://github.com/lingui/js-lingui/pull/1622/files#r1178201302

More details in documentation: https://tstyche.org/reference/expect-api#type-widening